src/engine/renderer/tr_init.cpp

```diff
@@ -219,6 +219,8 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 	cvar_t      *r_showNormalMaps;
 	cvar_t      *r_showMaterialMaps;
 	Cvar::Cvar<bool> r_showCubeProbes( "r_showCubeProbes", "Show static reflections cubemap placement", Cvar::NONE, false );
+	Cvar::Cvar<int> r_showCubeProbeFace( "r_showCubeProbeFace", "Render from the perspective of a selected static reflection "
```

Comment:
I tried it and it appears to draw the tiny 32x32 image in the lower left corner and stop rendering anything else. Is that what it's supposed to do? It's annoying that I have to type the command blindly to turn it back off (since the console is not rendered).

Reply:
Yes, since it changes the viewport to the cubemap's size. This is fixed now though.